app/ui/src/app/customizations/tech-extensions/tech-extension-delete-modal.component.ts

```diff
@@ -34,26 +34,20 @@ export class TechExtensionDeleteModalComponent implements OnInit, OnDestroy {
     this.modalService.show(this.id).then( modal => {
       if (modal.result) {
         this.extensionStore.delete(this.extension).subscribe(ext => {
-          this.notificationService.message(
-            NotificationType.SUCCESS,
-            'Deleted!',
-            'The technical extension "' + this.extension.name + '" has been deleted',
-            false,
-            null,
-            [],
-          );
+          this.notificationService.popNotification({
+            type: NotificationType.SUCCESS,
+            header: 'Deleted!',
+            message: 'The technical extension "' + this.extension.name + '" has been deleted'
```

Comment:
I'd use template strings instead:

message: `The technical extension ${this.extension.name} has been deleted`